app/src/main/java/me/hufman/androidautoidrive/L.kt

```diff
@@ -53,6 +53,7 @@ object L {
 	var MUSIC_TURN_REPEAT_ALL_ON = "Turn Repeat All On"
 	var MUSIC_TURN_REPEAT_ONE_ON = "Turn Repeat One On"
 	var MUSIC_TURN_REPEAT_OFF = "Turn Repeat Off"
+	var MUSIC_TEMPORARY_PLAYLIST_DESCRIPTION = "AndroidAutoIDrive dummy playlist - please do not delete"
```

Comment:
Could this match the strings.xml value, just for cleanliness? The only place it would possibly be seen is in unit tests, but just for cleanliness' sake!

Reply:
Yeah definitely, sorry I missed changing that string. Updated to match the strings.xml equivalent key